tests/005-test-devmapper-cleanup

```diff
@@ -0,0 +1,61 @@
+source $SRCDIR/libtest.sh
+
+# Test DEVS= directive. Returns 0 on success and 1 on failure.
```

Comment:
This comment needs to be changed to reflect that reset operation is being tested.